accepted/2025/dotnet-pack-nuspec.md

```diff
@@ -0,0 +1,75 @@
+
+### Options Relevant to `.nuspec`-only Support in `dotnet pack`
+
+| Option / Flag                  | Description                                                      | Notes / Status for .nuspec-only                                 | Comparison (`nuget pack -h` / `dotnet pack -h`)                |
+|------------------------------- |------------------------------------------------------------------|-----------------------------------------------------------------|---------------------------------------------------------------|
+| `--property` / `-p:`           | Set or override a property for token replacement in `.nuspec`.   | Reused; interpret for token replacement, not just MSBuild.       | `-Properties` in `nuget pack`, `--property` in `dotnet pack`   |
+| `--output`                     | Specifies the output directory for the generated package.         | Reused; already available.                                       | `-OutputDirectory` in `nuget pack`, `--output` in `dotnet pack`|
+| `--basepath`                   | Specifies the base path for resolving relative file paths.        | New; needed for file path resolution in `.nuspec`.               | `-BasePath` in `nuget pack`, not present in `dotnet pack`      |
+| `--version-suffix` / `--version` | Sets the version or version suffix for the package.             | Reused/Clarify; clarify how it applies to `.nuspec` versioning.  | `-Version` in `nuget pack`, `--version-suffix` in `dotnet pack`|
+| `--exclude`                    | Exclude specific files or patterns from the package.              | New; needed for file exclusion support.                          | `-Exclude` in `nuget pack`, not present in `dotnet pack`       |
+| `--nologo`                     | Suppresses display of the startup banner and copyright.           | Reused; already available.                                       | `-NoLogo` in `nuget pack`, `--nologo` in `dotnet pack`         |
+| `--min-client-version`         | Sets the minimum NuGet client version required to install the package. | Consider; may need to add.                                 | `-MinClientVersion` in `nuget pack`, not in `dotnet pack`      |
+| `--no-package-analysis`        | Skips package analysis after building the package.                | Consider; may need to add.                                       | `-NoPackageAnalysis` in `nuget pack`, not in `dotnet pack`     |
+| `--no-default-excludes`        | Disables default file exclusion patterns.                         | Consider; may need to add.                                       | `-NoDefaultExcludes` in `nuget pack`, not in `dotnet pack`     |
+| `--serviceable`                | Marks the package as serviceable.                                 | Consider; may need to add or clarify.                            | `-Serviceable` in `nuget pack`, `--serviceable` in `dotnet pack`|
```

Comment:
I'm very confident servicable is settable in the nuspec. I see no need for it to be a CLI option